me.sergiotarxz.JapaChar.yml

```diff
@@ -0,0 +1,99 @@
+id: "me.sergiotarxz.JapaChar"
+branch: "main"
+runtime: "org.gnome.Platform"
+runtime-version: "46"
+sdk: "org.gnome.Sdk"
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-config/.japachar
```

Comment:
why do you need this? In general accessing `xdg-config` in incorrect and not allowed.

Reply:
To preserve progress of users between reinstallations.

Reply:
Are you sure that's the right option.

you probably want `--persist=.japachar` instead.

Ideally it should follow xdg standard and use `$XDG_CONFIG_HOME` to locate the directory.

Reply:
I will try that, thank you, it is my first time doing this.

Reply:
so you didn't test it.....

Reply:
Done.

Reply:
@hfiguiere Yes, I tested with flatpak-builder and running the app.